net/knock/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        jvinet knock 258a27e5a47809f97c2b9f2751a88c2f94aae891
+version             0.7.8-20151227
+#name               knock
+categories          net
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-2
+description         A port-knocking implementation
+long_description    ${description}
```

Comment:
Please expand this. Use the long description from the homepage:

`knockd is a port-knock server. It listens to all traffic on an ethernet (or PPP) interface, looking for special "knock" sequences of port-hits. A client makes these port-hits by sending a TCP (or UDP) packet to a port on the server. This port need not be open -- since knockd listens at the link-layer level, it sees all traffic even if it's destined for a closed port. When the server detects a specific sequence of port-hits, it runs a command defined in its configuration file. This can be used to open up holes in a firewall for quick access.
`